pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
 
     # Multimodal
     "Pillow>=9.4.0",
+    "math-verify[antlr4_9_3]"
```

Comment:
Can you put this in the ``async_rl`` requirements rather than the main requirements?

Reply:
Done!